packages/nodejs/src/client.ts

```diff
@@ -621,20 +595,16 @@ export default class ConcordiumNodeClient {
         if (!isValidHash(blockHash)) {
             throw new Error('The input was not a valid hash: ' + blockHash);
         }
-        const blockHashObject = new BlockHash();
-        blockHashObject.setBlockHash(blockHash);
-
-        const response = await this.sendRequest(
-            this.client.getInstances,
-            blockHashObject
-        );
+        const input: BlockHash = { blockHash };
+        const { value } = await this.client.getAnonymityRevokers(input)
```

Comment:
I don't think this is right

Reply:
Good catch, I'll check them all again for good measure 😄